src/xunit.netcore.extensions/Discoverers/ConditionalClassDiscoverer.cs

```diff
@@ -53,12 +53,7 @@ internal static bool EvaluateParameterHelper(IAttributeInfo traitAttribute)
                 if (string.IsNullOrWhiteSpace(entry)) continue;
 
                 MethodInfo conditionMethodInfo = ConditionalTestDiscoverer.LookupConditionalMethod(calleeType, entry);
-                if (conditionMethodInfo == null)
-                {
-                    // Unable to get MethodInfo. It's caused by bad user input.
-                    // We need to report some error here. For now, just don't run the test.
-                    return false;
-                }
+                Debug.Assert(conditionMethodInfo != null, $"Unable to get MethodInfo, please check input for {entry}.");
```

Comment:
I don't know if this will work as we might be using the release build of xunit extensions.

Reply:
Ok, I searched through Xunit repo and not able to find a proper API which we can use to report an error. We can throw a Exception here.